PurchasesCoreSwift/IntroEligibilityCalculator.swift

```diff
@@ -39,26 +40,34 @@ import StoreKit
         }
         do {
             let receipt = try receiptParser.parse(from: receiptData)
-            let purchasedProductIdsWithIntroOffersOrFreeTrials = receipt.purchasedIntroOfferOrFreeTrialProductIdentifiers()
```

Comment:
124 character width ugh. Maybe we should consider moving to the 2-space indent instead of 4?

Reply:
@aboedo @beylmk thoughts?

Reply:
i wouldn't either

Reply:
happy to give it a shot. If we want to do the transition, though, we need to do it as a single PR since it involves a lot of code, we should try to not mix it with any other changes. 

The one part that I don't love is that we need to start requiring external devs to set their xcodes to 2 spaces for PRs

Reply:
Yeah, that's the more annoying part. We could use fastlane to:
- install a pre-commit hook that runs the linter.
- bootstrap the project to install the linter

is the trade-off worth it? I'm not totally sure, but if we do start using SwiftLint, we probably will want to bootstrap people anyway with the linter, so the infrastructure will be there.

Reply:
I love the idea of installing a pre-commit hook that runs the linter. I was actually wondering how to do that and share the pre-commit hook with the contributors. We can use that for Android's detekt too. I think I will leave that for another PR to always be shipping :love_you_gesture:

As @aboedo said, maybe better to do the space changes in another PR when we don't have any other big PR open so there's no many conflicts.

Reply:
I'm happy for you to make these changes and I can rebase my PR if you'd like. Totally up to you, though.

Reply:
Go for it! You've definitely got more experience with Ruby than I do 🥇